.github/CODEOWNERS

```diff
@@ -2,36 +2,36 @@
-/app/services/search/                               @thepracticaldev/sre
-/app/views/articles/                                @thepracticaldev/hells-bells
-/app/views/internal/                                @thepracticaldev/snackcase
-/app/views/moderation/                              @thepracticaldev/snackcase
-/app/views/notifications/                           @thepracticaldev/delightful
-/app/views/partnerships/                            @thepracticaldev/cool
-/app/views/stories/                                 @thepracticaldev/hells-bells
-/app/workers/                                       @thepracticaldev/sre
-/config/                                            @thepracticaldev/sre
-/db/                                                @thepracticaldev/sre
-/docs/                                              @thepracticaldev/oss    @jacobherrington
-/lib/data_update_scripts/                           @thepracticaldev/sre
-/lib/sidekiq/                                       @thepracticaldev/sre
-/spec/rails_helper.rb                               @thepracticaldev/sre
-/spec/requests/internal/                            @thepracticaldev/snackcase
-/spec/support/                                      @thepracticaldev/sre
-/spec/system/internal/                              @thepracticaldev/snackcase
-Gemfile                                             @thepracticaldev/sre    @thepracticaldev/oss
-Gemfile.lock                                        @thepracticaldev/sre    @thepracticaldev/oss
-package.json                                        @thepracticaldev/oss    @nickytonline
-yarn.lock                                           @thepracticaldev/oss    @nickytonline
-.travis.yml                                         @thepracticaldev/sre
+/*.md                                               @forem/oss
+/app/assets/stylesheets/internal/                   @forem/snackcase
+/app/controllers/articles_controller.rb             @forem/hells-bells
+/app/controllers/internal/                          @forem/snackcase
+/app/services/notifications/                        @forem/delightful
+/app/controllers/notifications/                     @forem/delightful
+/app/workers/notifications/                         @forem/delightful
+/app/controllers/internal/broadcasts_controller.rb  @forem/delightful
+/app/controllers/async_info_controller.rb           @forem/sre
+/app/controllers/pages_controller.rb                @forem/cool
+/app/controllers/stories_controller.rb              @forem/hells-bells
+/app/services/search/                               @forem/sre
+/app/views/articles/                                @forem/hells-bells
+/app/views/internal/                                @forem/snackcase
+/app/views/moderation/                              @forem/snackcase
+/app/views/notifications/                           @forem/delightful
+/app/views/partnerships/                            @forem/cool
```

Comment:
@forem/cool no longer exists, who should this be assigned to?

Reply:
I think no one. I would remove this line